include/zephyr/posix/time.h

```diff
@@ -77,6 +77,10 @@ extern "C" {
 #define TIMER_ABSTIME 4
 #endif
 
+#ifndef DELAYTIMER_MAX
+#define DELAYTIMER_MAX 20
```

Comment:
It might make more sense to create a Kconfig option for this rather than hard-coding it. 
e.g.
```
if TIMER

config POSIX_TIMER_DELAYTIMER_MAX
	int "Maximum count returned by timer_getoverrun()"
	default 20
	help
	  ...

endif
```

Reply:
> If you remove the #ifndef, does it break anything? That would be better if possible.

It does not break anything.